packages/client/src/views/RegisterForm/duplicate/DuplicateFormTabs.tsx

```diff
@@ -702,7 +700,11 @@ export const DuplicateFormTabs = (props: IProps) => {
           leftValue: (
             <Text variant="reg16" element="span" color="grey600">
               {actualRegData.status
-                ? intl.formatMessage(regStatusMessages[actualRegData.status])
+                ? intl.formatMessage(
+                    regStatusMessages[
+                      actualRegData.status as unknown as RegStatus
+                    ]
```

Comment:
I prefer changing the status's original type to `RegStatus` [here](https://github.com/opencrvs/opencrvs-core/blob/d46f47e0dbe6cafacd5a03d4d7df6b198849190d/packages/client/src/declarations/index.ts#L228) instead of this type assertion.

You'll need to change https://github.com/opencrvs/opencrvs-core/blob/d46f47e0dbe6cafacd5a03d4d7df6b198849190d/packages/client/src/declarations/index.ts#L1559-L1563 these lines as well to something like 
```
      const downloadedAppStatus: RegStatus | undefined =
        eventData &&
        eventData.registration &&
        eventData.registration.status &&
        eventData.registration.status[0].type
```

, but it would be a nice improvement to the technical debt we have in the front end.

Reply:
Thanks man, I wasn't sure whether to avoid touching other part of the code as much as possible but I get your point, thinking we could deal with the tech debt as a separate ticket, would you agree?

I'm honestly a bit confused regarding `downloadedAppStatus: RegStatus | undefined` does this mean the type of `downloadedAppStatus` is of `RegStatus`  always? I'm asking because I saw the [DOWNLOAD_STATUS enum](https://github.com/opencrvs/opencrvs-core/blob/1854f16ae65f9308cb4caf5753fcace65a061e13/packages/client/src/declarations/index.ts#L152-L160) with the values below, are they not related?
```
export enum DOWNLOAD_STATUS {
  READY_TO_DOWNLOAD = 'READY_TO_DOWNLOAD',
  DOWNLOADING = 'DOWNLOADING',
  DOWNLOADED = 'DOWNLOADED',
  FAILED = 'FAILED',
  FAILED_NETWORK = 'FAILED_NETWORK',
  READY_TO_UNASSIGN = 'READY_TO_UNASSIGN',
  UNASSIGNING = 'UNASSIGNING'
}
```
I'm avoiding a case of effort spent on refactors bloating up the effort required to do the fix.

Reply:
>Thanks man, I wasn't sure so I avoided touching other parts of the codebase as much as possible but I get your point, my thinking was, we could deal with the tech debt as a separate ticket. I'm avoiding a case of effort spent on refactors bloating up the effort required to do the fix. would you agree?

Thanks for explaining your thought process, I see you 👍🏻 In our team refactors in pull requests like these are good, unless it's specifically an urgent bug fix. In my opinion, feel free to always apply the 1% improvement to the pull requests :)